bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -31,7 +36,16 @@ export default class Chat extends Component {
         <header className={styles.header}>
           <Link className={styles.closeChat} to="/users">
             <Icon iconName="left-arrow" /> {title}
+            <Button className={styles.closeBtn}
```

Comment:
Could you separate the code for this feature from this pr?
I don't think this has anything to do with hiding the cam button.

About this close button, this is a feature that is a little complex with our current implementation of the open chats list. As far as I remember the idea is that this close button will remove the chat from the open chats list, in the user list.